pkg/cluster/internal/kubeadm/config.go

```diff
@@ -278,6 +287,20 @@ metadata:
   name: config
 `
 
+// JoinConfigTemplate is the kubadm join config template
+const JoinConfigTemplate = `# config generated by kind
+# no-op entry that exists solely so it can be patched
+apiVersion: kubeadm.k8s.io/{{ .KubedamAPI }}
```

Comment:
Joins are done with the command line, i took one option but I'm missing lot of context, appreciate any feedback and advice

Reply:
i think we can pass the same kubeadm.cfg to both init and join (with the multi-yaml doc; `---` separated). i was under the impression we are already doing that.

Reply:
I abandoned that idea because I thought that will break the `kubeadm join` and it could simplify the join file creation since it needs fewer parameters, any way you're the expert here